<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Poker_Client</class>
 <widget class="QWidget" name="Poker_Client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PokerClient</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser_log">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>291</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_lelaoji">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>291</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_lelaoji">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_send">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>470</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ready">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>410</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>准备</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_500">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>331</x>
     <y>231</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>+500</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_pass">
   <property name="geometry">
    <rect>
     <x>458</x>
     <y>231</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>过</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_giveup">
   <property name="geometry">
    <rect>
     <x>585</x>
     <y>231</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>弃牌</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_join">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>520</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>加入房间</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>432</x>
     <y>492</y>
     <width>96</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入你的名字：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="geometry">
    <rect>
     <x>534</x>
     <y>492</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
